sspl_test/rabbitmq_start_checker

```diff
@@ -29,56 +29,19 @@ import random
 test_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 os.sys.path.append(os.path.join(test_path))
 from sspl_test.framework.utils.conf_utils import Conf, SSPL_TEST_CONF
+from cortx.utils.message_bus import MessageBus
```

Comment:
rename the file too to avoid rmq name in it

Reply:
Done